lib/core/config.js

```diff
@@ -140,6 +142,21 @@ Config.prototype.loadContractsConfigFile = function() {
   this.contractsConfig = this._mergeConfig(configFilePath, configObject, this.env);
 };
 
+Config.prototype.loadExternalContractsFiles = function() {
+  let contracts = this.contractsConfig.contracts;
+  for (let contractName in contracts) {
+    let contract = contracts[contractName];
+    if (!contract.file) {
+      continue;
+    }
+    if (fs.existsSync(contract.file)) {
```

Comment:
I think there should be an `else` to log a warning in case the file doesn't exist. Something like "Tried to locate file X, but it isn't in the directory, nor in the node_modules."

Reply:
agree. added a else and log the error